### DISCUSSION TITLE
Filter out long fields in __repr__

### DISCUSSION THREAD

**Discussion by bverhoeve** [pydantic/pydantic#2604] [Question] [answered]
Hi, thanks for the great work on Pydantic!

I'm currently working with a Pydantic model that has a field that is a base-64 encoded string of some data. When calling `__repr__` to print out an instance of the object, the default Pydantic `__repr__` method will print out every field, resulting in a bit of an unreadable string when printed out.

I've looked into the source code and saw that the `__repr_args__` method is being used to construct the representation string, but haven't been able yet to filter out the field that's been giving me trouble.

Is there a way to either mark that the field should not be included in `__repr_args__`? Or is there a way to filter the field out by overriding the `__repr__` method? Thanks!

**Comment by PrettyWood** [comment, marked as answer]
HI @bverhoeve 
It may well be added in v1.9 :) https://github.com/samuelcolvin/pydantic/pull/2593

**Reply by bverhoeve** [reply]
Hi @PrettyWood 
Cool! I'll keep an eye out for the next release 😄

**Reply by GaniMarquez** [reply]
Hi @PrettyWood 

Would it be okay to know your roadmap regarding #2593?
We are currently using Pydantic right now and it would be great to have that feature available so that we can utilize it as we design our models.

Thanks!
